sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/QuerySeriesRequestOptions.cs

```diff
@@ -22,6 +22,12 @@ public class QuerySeriesRequestOptions : QueryRequestOptions
         /// </summary>
         public IDictionary<string, TimeSeriesVariable> InlineVariables { get; }
 
+        /// <summary>
+        /// The maximum number of property values in the whole response set, not the maximum number of property values per page.
+        /// Defaults to 10,000 when not set. Maximum value of take can be 250,000.
+        /// </summary>
+        public int? MaximumNumberOfEvents { get; set; }
```

Comment:
Is this something only applicable to Events and Series? Or can it be in the base class?

Reply:
Only for events and series. Not applicable for Aggregate Series